packages/webdriverio/src/utils.js

```diff
@@ -61,6 +61,11 @@ export const findStrategy = function (value, isW3C, isMobile) {
         using = '-android uiautomator'
         value = value.slice(8)
 
+    // recursive element search using the DataMatcher library (Android only)
+    } else if (value.indexOf('datamatcher=') === 0) {
```

Comment:
if you add one more to `DIRECT_SELECTOR_REGEXP` then it `else if` is redundant.

It will be caught by this if:
```js
    const match = value.match(DIRECT_SELECTOR_REGEXP)
    if (match) {
        /**
         * ensure selector strategy is supported
         */
        if (!isMobile && isW3C && !W3C_SELECTOR_STRATEGIES.includes(match[1])) {
            throw new Error('InvalidSelectorStrategy') // ToDo: move error to wdio-error package
        }

        return {
            using: match[1],
            value: match[2]
        }
    }
```